Library/ViewModels/PledgePaymentPlansOptionViewModel.swift

```diff
@@ -38,13 +64,35 @@ public final class PledgePaymentPlansOptionViewModel:
       }
 
     self.titleText = configData.map { getTitleText(by: $0.type) }
-    self.subtitleText = configData.map { getSubtitleText(by: $0.type) }
+    self.subtitleText = configData
+      .map { getSubtitleText(by: $0.type, isSelected: $0.selectedType == $0.type) }
     self.subtitleLabelHidden = self.subtitleText.map { $0.isEmpty }
 
     self.notifyDelegatePaymentPlanOptionSelected = self.optionTappedProperty
       .signal
       .withLatest(from: configData)
       .map { $1.type }
+
+    let isPledgeOverTimeAndSelected = configData.map {
+      $0.type != .pledgeOverTime || $0.type != $0.selectedType
```

Comment:
This statement is a little confusing, should it be `$0.type == .pledgeOverTime && $0.type == $0.selectedType`?

Reply:
Good catch, Amy! I think this initially had other conditions, but now your suggestion makes perfect sense. I’ll apply your recommendation as is.